### PR TITLE
Add pull request and issue templates

### PR DESCRIPTION
**Pull request by robobario**
Inspired (copy + paste) by the Strimzi project templates, thanks!

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+1. Configure kroxylicious like this ...
+2. Run command '....'
+3. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**YAML files and logs**
+
+Attach or copy paste your kroxylicious configuration YAML.
```
Comment (SamBarker) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: Should we recommend removing cluster addresses etc?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+_Please use this to only for bug reports. For questions or when you need help, you can use the [GitHub Discussions](https://github.com/kroxylicious/design/discussions)
```
Comment (SamBarker) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]: should there be a second `_` somewhere maybe after `reports`?

**Review by SamBarker — APPROVED** [review]
LGTM

Does this have to be configured per repo or can we configure at the org level?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (k-wall) [reply .github/ISSUE_TEMPLATE/bug_report.md:10]: Is design discussions the right place?   I would have thought a question discussion in this repo.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Questions and Discussions
+    url: https://github.com/strimzi/strimzi-kafka-operator/discussions
```
Comment (k-wall) [inline .github/ISSUE_TEMPLATE/config.yml:4]: Strimzi ref.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (robobario) [reply .github/ISSUE_TEMPLATE/bug_report.md:10]: Added the other `_` and pointed both this and the issue config at `https://github.com/kroxylicious/kroxylicious/discussions`

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (robobario) [reply .github/ISSUE_TEMPLATE/bug_report.md:26]: Added a bit about redacting sensitive info

File: .github/ISSUE_TEMPLATE/config.yml (hunk 3 above)
Comment (robobario) [reply .github/ISSUE_TEMPLATE/config.yml:4]: fixed

**Comment by robobario** [comment]
> Does this have to be configured per repo or can we configure at the org level?

Looks like you can do it at an org level with a repository called `.github`

https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file

**Comment by robobario** [comment]
opened https://github.com/kroxylicious/.github/pull/1 as an alternative

**Review by SamBarker — APPROVED** [review]
Given the inability to have the org wide templates point at the local repository for discussions etc I'm leaning towards having them per repo. 

Thanks for looking at both options @robobario

**Pull request merged**